utility/actor.go

```diff
@@ -0,0 +1,484 @@
+package utility
+
+import (
+	"github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/shared/types"
+	typesGenesis "github.com/pokt-network/pocket/shared/types/genesis"
+	typesUtil "github.com/pokt-network/pocket/utility/types"
+	"math"
+	"math/big"
+)
+
+// 'Actor' is the consolidated term for common functionality among the following network actors: app, fish, node, val
+// Within this file, there are all the state based CRUD operations that are shared between these abstractions.
+// The ideology of the separation of the actors is based on the expectation of actor divergence in the near future.
+// The current implementation attempts to simplify code footprint and complexity while enabling future divergence.
+// It is important to note, that as production approaches, the idea is to attempt more consolidation at an architectural
+// multi-module level. Until then, it's a fine line to walk.
+
+// setters
+
+func (u *UtilityContext) SetActorStakedTokens(address []byte, tokens *big.Int, actorType typesUtil.ActorType) types.Error {
```

Comment:
NIT: Can you make `actorType typesUtil.ActorType` the first param so it's easier to read during usage?

Ditto below